client/my-sites/theme/theme-related-themes-card/index.jsx

```diff
@@ -0,0 +1,86 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import i18n from 'lib/mixins/i18n';
+import SectionHeader from 'components/section-header';
+import { getDetailsUrl } from 'my-sites/themes/helpers';
+
+var ThemeRelatedThemesCard = React.createClass( {
+
+	propTypes: {
+		currentTheme: React.PropTypes.string.isRequired
+	},
+
+	getRelatedThemes() {
+		var themes = [
+			{ name: 'Twenty Sixteen', slug: 'twentysixteen' },
+			{ name: 'Rowling', slug: 'rowling' },
+			{ name: 'Hemingway Rewritten', slug: 'hemingway-rewritten' },
+			{ name: 'Gazette', slug: 'gazette' },
+			{ name: 'Ecto', slug: 'ecto' },
+			{ name: 'Isola', slug: 'isola' },
+			{ name: 'Edin', slug: 'edin' },
+			{ name: 'Sela', slug: 'sela' },
+			{ name: 'Pique', slug: 'pique' },
+			{ name: 'Harmonic', slug: 'harmonic' },
+		];
+
+		var selectedThemes = [];
+
+		while( selectedThemes.length < 2 ) {
+			let randomThemeIndex = Math.floor(Math.random() * themes.length );
+			let theme = themes.splice( randomThemeIndex, 1 )[0];
+			// Check if it is not current theme, if it is than discard
```

Comment:
I wonder if making `themes` an object instead of an array, and `delete`ing `currentTheme` from it would be tidier, since there would be less to do inside this loop?